tests/integration/EntitySerializationRoundtripTest.php

```diff
@@ -35,41 +35,37 @@ public function testEntitySerializationRoundtrips( Entity $entity ) {
 	}
 
 	public function entityProvider() {
-		$entities = array();
+		$items = array();
```

Comment:
This is still an `entityProvider`, so `$entities` seems like a better name... just noticed there is actually a property put in this thing at the end of the function :)